lib/modules/orangered.js

```diff
@@ -171,9 +166,32 @@ const setupFaviconBadge = _.once(() => {
 	return favicon;
 });
 
-function setupFloatingButtons() {
-	if (!module.options.showFloatingEnvelope.value) return;
-	if ($floatingInboxCount) return;
+const orangeredElements = () => ({
+	...nativeButtons(),
+	...floatingButtons(),
+	$modmailButton: $('#modmail'),
+});
+
+const $orangeredElements = () => {
+	const dict = orangeredElements();
+	const elements = Object.keys(dict).map(key => dict[key]);
+	return elements.reduce(($all, ele) => $all.add(ele), $());
+};
```

Comment:
@erikdesjardins How's this read? Feels a little awkward, but it's really only called once.